src/main/java/com/example/SOPSbackend/configuration/CorsConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.example.SOPSbackend.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class CorsConfig {
+
+    @Bean
+    public WebMvcConfigurer corsConfigurer(){
```

Comment:
Trzeba sie dowiedziec jakie to ma konsekwencje.

Reply:
Wygląda na to, że bardzo dobre rozwiązanie.